scripts/package-build/vpp/package.toml

```diff
@@ -15,6 +15,8 @@ apply_patches = false
 pre_build_hook = """
 mkdir -p ../patches/vpp/
 rsync -av ../vyos-vpp-patches/patches/vpp/ ../patches/vpp/
+git clone https://github.com/sflow/vpp-sflow ../vpp-sflow
```

Comment:
Can we use an additional `package` instead of cloning? After the `vyos-vpp-patches`
```
[[packages]]
name = "vpp-sflow"
commit_id = "v0.9.02-2"  # tag or commit id
scm_url = "https://github.com/sflow/vpp-sflow"
build_cmd = "/bin/true"
apply_patches = false
```

This way we can get a specific commit id/tag and rebuild it properly if `commit_id` is changed.